src/index.html

```diff
@@ -3,10 +3,345 @@
+        
+          <div class="card__rating">  
+            <div class="card__stars stars stars--4">
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+        
+          <p class="card__price">
+            Price:
+            <span class="card__price-in-money">
+              $2,199
+            </span>
+          </p>
+          
+          <a 
+            class="card__button" 
+            href="BUY" 
+          >
+            BUY
+          </a>
+        </div>
+        
+        <div class="card">
+          <img 
+          class="card__logo" 
+          src="images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+          <h1 class ="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <p class="card__code">Product code: 195434</p>
+        
+          <div class="card__rating">  
+            <div class="card__stars stars stars--4">
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+        
+          <p class="card__price">
+            Price:
+            <span class="card__price-in-money">
+              $2,199
+            </span>
+          </p>
+          
+          <a 
+            class="card__button" 
+            href="BUY" 
+          >
+            BUY
+          </a>
+        </div>
+
+        <div class="card">
+          <img 
+          class="card__logo" 
+          src="images/imac.jpeg" 
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+          <h1 class ="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <p class="card__code">Product code: 195434</p>
+        
+          <div class="card__rating">  
+            <div class="card__stars stars stars--4">
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+              <div class="stars__star star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+        
+          <p class="card__price">
+            Price:
+            <span class="card__price-in-money">
+              $2,199
+            </span>
+          </p>
+          
+          <a 
+            class="card__button" 
+            href="BUY" 
```

Comment:
If you click on link now it sends user to non-existing page 
```suggestion
            href="#" 
```